CHANGES.md

```diff
@@ -1,3 +1,21 @@
+## edge-19.7.1
+
+* CLI
+  * Added more descriptive output to the `linkerd check` for control plane
```

Comment:
the `linkerd check` **command** ?

Reply:
I think this should be "the linkerd check _output_"?